### PR TITLE
Dh apmspii 526 remove content location check from polling loop

### PR DESCRIPTION
**Pull request by DominicH247**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: docker/async-slowapp/src/handlers.js [hunk 1]
```diff
@@ -262,7 +262,7 @@ async function delete_poll(req, res, next) {
     respond(req, res, next, 200);
 }
 
-
+let poll_counter = 0;
```
Comment (vurt007) [inline docker/async-slowapp/src/handlers.js:265]: no to making this stateful

File: docker/async-slowapp/src/handlers.js [hunk 2]
```diff
@@ -273,7 +273,7 @@ async function poll(req, res, next) {
     let poll_count = cookies['poll-count'];
 
     if (poll_count !== undefined) {
-        poll_count = (parseInt(poll_count.split('=')[1]) + 1).toString();
+        poll_counter++
```
Comment (vurt007) [inline docker/async-slowapp/src/handlers.js:276]: the cookie was incremented, in order to test that cookies were read a written back correctly ...

File: docker/async-slowapp/src/handlers.js [hunk 3]
```diff
@@ -291,7 +291,7 @@ async function poll(req, res, next) {
     let resp_headers = new MultiValueHeaders();
 
     if (poll_count !== undefined) {
-        resp_headers.withNewCookies({"poll-count": `poll-count=${poll_count}`}, 'set-cookie');
+        resp_headers.withNewCookies({"poll-count": `poll-count=${poll_counter}`}, 'set-cookie');
```
Comment (vurt007) [inline docker/async-slowapp/src/handlers.js:294]: again .. this was using an explicit value tracked by the client cookie

**Review by vurt007 — CHANGES_REQUESTED** [review]

**Review by vurt007 — COMMENTED** [review]
would be good to see an explicit test where only the initial content location returns a 'content-location' header too

File: docker/async-slowapp/src/handlers.js (hunk 2 above)
Comment (DominicH247) [reply docker/async-slowapp/src/handlers.js:276]: @vurt007 thanks for the feedback. I've mis understood what this was doing, I'll change it back

**Review by vurt007 — APPROVED** [review]

**Pull request merged**